src/main/java/uk/gov/hmcts/reform/pip/account/management/service/InactiveAccountManagementService.java

```diff
@@ -14,6 +14,8 @@
 @Service
 public class InactiveAccountManagementService {
 
+    private static final int SSO_ADMIN_ACCOUNT_SIGN_IN_NOTIFICATION_DAYS = 0;
```

Comment:
I'm thinking it might be cleaner than rather having 0 here / checking for it in the SQL query, explicitally having two separate repository methods for the notification + deletion, so the notification only picks up PI_AAD users, would be better.

What do you think?

Reply:
Split out the notification and deletion queries